src/pages/ListScholarships.jsx

```diff
@@ -119,16 +128,39 @@ function ListScholarships() {
     });
 
   return (
-    <Container>
+    <Box sx={{ display: 'flex', position: 'relative', height: '100%' }}>
       <Helmet>
         <title>{t('listScholarships.titleTag')}</title>
       </Helmet>
-      <Typography variant="h4" component="h1" align="center" style={{ p: 1 }}>
-        {t('general.scholarships')}
-      </Typography>
-      <FilterBar queryParams={params} {...{ setQueryParam }} />
-      <ScholarshipList listFn={listScholarships} />
-    </Container>
+      <Hidden mdDown>
```

Comment:
looks like `Hidden` is [deprecated](https://mui.com/guides/migration-v4/#hidden)